### PR TITLE
Интерфейс по комиксам

### PR DESCRIPTION
**Pull request by Zoldrek**

### CODE REVIEW

File: interface/interface comic/search_comics_with_featurnig_two_characters/kb/ui_components/search_comics_with_featurnig_two_characters_component/ui_search_comics_with_featurnig_two_characters_component_load.scs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ui_search_comics_with_featurnig_two_characters_in_memory
+	<- ui_scgg_control; sc_node_not_relation;
```
Comment (MikhailSadovsky) [inline interface/interface comic/search_comics_with_featurnig_two_characters/kb/ui_components/search_comics_with_featurnig_two_characters_component/ui_search_comics_with_featurnig_two_characters_component_load.scs:2]: Please update to "ui_scg_control"

File: interface/interface comic/search_comics_with_featurnig_two_characters/sc-web/components/search_comics_with_featurnig_two_characters_component/src/search_comics_with_featurnig_two_characters_component.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    $('#' + sandbox.container).prepend('<div class="inputBox" id="series-tools-' + sandbox.container + '"></div>');
+    $('#series-tools-' + sandbox.container).load('static/components/html/search_comics_with_featurnig_two_characters_component.html', function () {
+        SCWeb.core.Server.resolveScAddr(keynodes, function (keynodes) {
+            SCWeb.core.Server.resolveIdentifiers(keynodes, function (idf) {
+                var buttonSearch = idf[keynodes['ui_search_comics_with_featurnig_two_characters_in_memory']];
+                $(buttonFind).html(buttonSearch);
+                $(buttonFind).click(function () {
+                    var firstString = $(inputFirst).val();
+                    var secondString = $(inputSecond).val();
+                    console.log("First " + firstString);
```
Comment (MikhailSadovsky) [inline interface/interface comic/search_comics_with_featurnig_two_characters/sc-web/components/search_comics_with_featurnig_two_characters_component/src/search_comics_with_featurnig_two_characters_component.js:28]: Please remove "console.log" statements all over the file

File: interface/interface comic/update_web.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (MikhailSadovsky) [inline interface/interface comic/update_web.sh:1]: This script file should be moved to "ostis-art/scripts/" folder.
Only part with build of your component should be left. Please, rename the script as it should relate to component.
It'll be great to update "install_ostis.sh" to call this script so your web component will be ready on startup.

**Comment by Zoldrek** [comment]
Pull request updated.

File: scripts/install_ostis.sh [hunk 4]
```diff
@@ -26,6 +26,7 @@ include_kpm()
   echo 'add_subdirectory(${SC_MACHINE_ROOT}/../../problem-solver/cxx ${SC_MACHINE_ROOT}/bin)' >> ./CMakeLists.txt
 	cd ./scripts
 	./make_all.sh
+        ./install_interface_comic.sh
```
Comment (MikhailSadovsky) [inline scripts/install_ostis.sh:29]: it's better to move this call in separate function:
- Create function 'install_interface_comic' and call this script inside.
- Add function call after include_kpm (line 48)

**Comment by Zoldrek** [comment]
Pull request updated.

**Pull request merged**